Combinatorial_map/include/CGAL/Dart.h

```diff
@@ -241,7 +241,7 @@ namespace CGAL {
     }
 
   protected:
-    /// Neighboors for each dimension +1 (from 0 to dimension).
+    /// Neighbours for each dimension +1 (from 0 to dimension).
```

Comment:
```suggestion
    /// Neighbors for each dimension +1 (from 0 to dimension).
```

Reply:
I found in total 248 times `neighbour` / `neighbours` and in / starting with upper case in 43 files so a few more to go.
During my corrections I saw that there were different usages of neighbor and / neighbour but I forgot to ask you (I know there is that the US-English version is neighbor and the UK-English is neighbour).
I'll correct them all.

Reply:
During the corrections I found it also:
- in the bibliography, I left these unchanged
- in variable names / function names, for the time being I left them unchanged, though the question is what to do function names as this might lead to incompatibilities (names of private variables and argument names could be changed without problem I think).
  The total overview is in: [neighbour_left.txt](https://github.com/CGAL/cgal/files/10029801/neighbour_left.txt)